client/blocks/login/style.scss

```diff
@@ -217,3 +217,11 @@
 .login__form-password.is-hidden {
 	display: none;
 }
+
+.login__jetpack-logo {
+	text-align: center;
```

Comment:
Is this necessary?

Reply:
It doesn't look like it is necessary since `login__form-header-wrapper` also applies `text-align: center;`. I'll go ahead and remove it. 👍